drivers/ethernet/eth_native_posix.c

```diff
@@ -587,3 +599,24 @@ LISTIFY(CONFIG_ETH_NATIVE_POSIX_INTERFACE_COUNT, PTP_INIT_FUNC, (), _)
 LISTIFY(CONFIG_ETH_NATIVE_POSIX_INTERFACE_COUNT, DEFINE_PTP_DEVICE, (;), _);
 
 #endif /* CONFIG_ETH_NATIVE_POSIX_PTP_CLOCK */
+
+#if defined(CONFIG_ETH_NATIVE_POSIX_DRV_NAME_FROM_CMD_LINE)
+void eth_native_posix_options(void)
```

Comment:
```suggestion
static void add_if_name_arg(void)
```
No need for this to not be static
& better to not use the same name for the function and variable